src/Schema/Tables/Deseases/DiseaseCategories.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[DiseaseCategories]
+(
+	[CategoryId] INT NOT NULL IDENTITY(1, 1),
+	[Name] NCHAR(100) NOT NULL,
```

Comment:
Maybe NVARCHAR would be better, because perfomance.
http://qaru.site/questions/3428/what-is-the-difference-between-char-nchar-varchar-and-nvarchar-in-sql-server